app/views/students/sped_referral.pdf.erb

```diff
@@ -1 +1,56 @@
-SPED Referral for <%= @student.first_name %>
\ No newline at end of file
+<hr>
+<p>
+  <h4>Somerville Public Schools - SPED referral packet</h4>
+  <ul style="list-style-type:none">
+    <li>Report date: <%= Date.today.strftime("%-m/%-d/%y") %></li>
+    <li>Generated by: <%= @current_educator.full_name %></li>
+    <li>Source: <%= @url %></li>
+  </ul>
+</p>
+<hr>
+<p>
+  <h4><%= "#{@student.first_name} #{@student.last_name}"%></h4>
+  <ul style="list-style-type:none">
+    <li>Language: <%= @student.home_language %></li>
+    <li>Disability: <%= @student.disability || "None"%></li>
+    <li>504 plan: <%= @student.plan_504 %></li>
+  </ul>
+</p>
+<p>
+  <h4>Current services:</h4>
+  <ul style="list-style-type:none">
+    <% @services.each do |service| %>
+      <li><%= "Since #{service.date_started.strftime("%m/%d/%Y")} - #{service.service_type.name}" %></li>
+    <% end %>
+  </ul>
+</p>
+<p>
+  <h4>Attendance record:</h4>
+  <ul style="list-style-type:none">
+    <li><%= "NOTE: #{@current_school_year} school year is still in progress" %></li>
+    <% @student_school_years.each do |year| %>
+      <li><%= "Year: #{year.name} - Absences: #{year.absences.count} - Tardies: #{year.tardies.count}" %></li>
+    <% end %>
+  </ul>
+</p>
+<p>
+  <h4>Behavioral incidents:</h4>
+  <% @discipline_incidents.each do |incident| %>
+    <ul style="list-style-type:none">
+      <li><%= "#{incident.occurred_at.strftime("%-m/%-d/%Y")} - #{incident.incident_code} - #{incident.incident_location}"%></li>
+      <li><%= incident.incident_description %></li>
+    </ul>
+  <% end %>
+</p>
+<p>
+  <h4>Academic assessments:</h4>
+  <% @student_assessments.each do |test| %>
+    <ul style="list-style-type:none">
+      <li><%= "#{test.date_taken.strftime("%-m/%-d/%Y")} - #{test.assessment.family} #{test.assessment.subject} - #{test.percentile_rank.try(:ordinalize) || 'Unknown'} percentile" %></li>
```

Comment:
Not all assessments use `percentile_rank`, you can see which types use which fields here: https://github.com/studentinsights/studentinsights/blob/master/app/models/student_assessment.rb#L14 (and if they have multiple, just grab all the ones that have to be `.present?`)